kuzzleio/kuzzle-admin-console
# Save sort filter in raw json mode

## What does this PR do ?

Save "sort" filter from a raw query in json mode.

### How should this be manually tested?

- open the admin console to see the content of a collection
- open "query options"
- switch to "raw json mode"
- add a query and sort filter like:
```json
{
  "query": {},
  "sort": {"foo": "asc"}
}
```
- click on search
- go into another collection
- go back to the first collection under the "raw json mode" search options
- sort should still be there

![a](https://user-images.githubusercontent.com/592088/55332348-7b7a8c80-5495-11e9-9fc7-baeac809cac5.png)
